cmd/modern/root/config/connection-strings.go

```diff
@@ -58,11 +58,11 @@ func (c *ConnectionStrings) run() {
 
 	// connectionStringFormats borrowed from "portal.azure.com" "connection strings" pane
 	var connectionStringFormats = map[string]string{
-		"ADO.NET": "Server=tcp:%s,%d;Initial Catalog=%s;Persist Security Info=False;User ID=%s;Password=%s;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=%s;Connection Timeout=30;",
-		"JDBC":    "jdbc:sqlserver://%s:%d;database=%s;user=%s;password=%s;encrypt=true;trustServerCertificate=%s;loginTimeout=30;",
-		"ODBC":    "Driver={ODBC Driver 18 for SQL Server};Server=tcp:%s,%d;Database=%s;Uid=%s;Pwd=%s;Encrypt=yes;TrustServerCertificate=%s;Connection Timeout=30;",
+		"ADO.NET": "Server=tcp:%s,%#v;Initial Catalog=%s;Persist Security Info=False;User ID=%s;Password=%s;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=%s;Connection Timeout=30;",
+		"JDBC":    "jdbc:sqlserver://%s:%#v;database=%s;user=%s;password=%s;encrypt=true;trustServerCertificate=%s;loginTimeout=30;",
+		"ODBC":    "Driver={ODBC Driver 18 for SQL Server};Server=tcp:%s,%#v;Database=%s;Uid=%s;Pwd=%s;Encrypt=yes;TrustServerCertificate=%s;Connection Timeout=30;",
 		"GO":      "sqlserver://%s:%s@%s,%d?database=%s;encrypt=true;trustServerCertificate=%s;dial+timeout=30",
-		"SQLCMD":  "sqlcmd -S %s,%d -U %s",
+		"SQLCMD":  "sqlcmd -S %s,%#v -U %s",
```

Comment:
>%#v  [](http://example.com/codeflow?start=27&length=4)

why `%#v` instead of `%v` ? these aren't intended to be represented as "go-syntax" friendly

Reply:
With `%v` I still get 1,435
```
git\go-sqlcmd>.\sqlcmd.exe create mssql --accept-eula
Downloading mcr.microsoft.com/mssql/server:latest
Starting mcr.microsoft.com/mssql/server:latest
Created context "mssql" in "C:\Users\apoorvdeshmukh\.sqlcmd\sqlconfig", configuring user account...
Disabled "sa" account (and rotated "sa" password). Creating user "apoorvdeshmukh"
Now ready for client connections on port 1,435

HINT:
  1. Open in Azure Data Studio: sqlcmd open ads
  2. Run a query:               sqlcmd query "SELECT @@version"
  3. Start interactive session: sqlcmd query
  4. View sqlcmd configuration: sqlcmd config view
  5. See connection strings:    sqlcmd config connection-strings
  6. Remove:                    sqlcmd delete


```

Reply:
what is the query output formatter doing to get it without the comma? I don't recall using the `#v` 

```
 8:51:04.45 e:\git\go-sqlcmd>cmd\modern\modern -Q "select 10000"

-----------
      10000
```

Reply:
We use `#v` in couple of places like [these](https://github.com/microsoft/go-sqlcmd/blob/930c2999c7692eb338bd3fd98ceb3d7c33464e70/pkg/sqlcmd/format.go#L227).
As per [documentation](https://pkg.go.dev/fmt) it only uses default format of underlying type but in our case the `x/text/message` pkg seems to add comma using the `number.Formatter` for the initialized language. `#v` prevents that by using go-syntax friendly value.